analysis/active_analyses/fn-add_analysis.R

```diff
@@ -31,11 +31,17 @@ add_analysis <- function(
     cut_points$vax_unvax
   )
 
+  ipw_used <- ifelse(
+    cohort == "unvax",
+    FALSE,
+    TRUE
+  )
```

Comment:
I think this could just be called `ipw`:
```suggestion
  ipw <- ifelse(
    cohort == "unvax",
    FALSE,
    TRUE
  )
```